lightcurve/alcdef.py

```diff
@@ -0,0 +1,84 @@
+from django.contrib import admin
+from django.db import models
+from django.http import HttpResponse
+
+class AlcdefWriter(models.Model):
```

Comment:
This is a standalone class and doesn't need to be a django model - in that case, you can just inherit from `object` (and remove the django imports)

`class AlcdefWriter(object):`

Reply:
Done